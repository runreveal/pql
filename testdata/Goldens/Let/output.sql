SELECT * FROM "StateCapitals" ORDER BY "State" ASC NULLS FIRST LIMIT 3;
