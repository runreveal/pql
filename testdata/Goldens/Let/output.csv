"State","StateCapital"
Alabama,Montgomery
Alaska,Juneau
Arizona,Phoenix
